CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
         ([#2069](https://github.com/Automattic/pocket-casts-android/pull/2069))
     *   Sleep timer: Set sleep time end of episode for multiple episodes
         ([#2097](https://github.com/Automattic/pocket-casts-android/pull/2097))
+    *   Enable toggling episode artwork separately in different contexts
```

Comment:
I can't access the last item in portrait. I also tried to scroll but I did not work 

![Screenshot_20240426_085248_Pocket Debug](https://github.com/Automattic/pocket-casts-android/assets/42220351/ba11e946-ca93-44c4-b0f1-7833147ea1d6)

Reply:
Fixed in 2ac2ddb442e883813797b3cded77b77d0fa3b570.